api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -848,6 +851,22 @@ public void updateVerifiedInstitutionalAffiliation_update() {
         .isEqualTo(verifiedInstitutionalAffiliation);
   }
 
+  @Test(expected = BadRequestException.class)
+  public void updateProfile_changeVerifiedInstitutionalAffiliationForbidden() {
+    config.featureFlags.requireInstitutionalVerification = true;
+
+    final VerifiedInstitutionalAffiliation affiliation = createVerifiedInstitutionalAffiliation();
+    createAccountRequest.getProfile().setVerifiedInstitutionalAffiliation(affiliation);
```

Comment:
I really only like to use class members for state that's not mutable, and create things like requests that have to be dynamic in the test methods themselves. As a reader, I don't know what side effects createUser() has, or what's happened to createAccountRequest in the meantime.